### ISSUE TITLE
Bigquery terraform clustering enablement using terraform

### ISSUE THREAD

**Issue by rmanktalacolt** [terraform-google-modules/terraform-google-bigquery#155]
### TL;DR

Hi,

I have got a scenario where in i need to apply clustering on few of the tables using terraforms. 
enabled the clusterting parameter and passed the json with below form to the table in main.tf..

Q1. Is the below syntax correct to pass the cluster name to main.tf

took the reference from : https://cloud.google.com/bigquery/docs/reference/rest/v2/tables

TABLENAME_CLUSTERING.json
{
  "fields": [
    "FIELD_NAME"
  ]
}

when deployed, there is no change on the table.

below is the syntax for the clusting parameters in tables in main.tf file 

clustering         = try(t.clustering && t.clustering == true ? jsondecode(file("${path.module}/${upper(t.id)}_CLUSTERING.json"))[0] : null, null),

is there anything wrong ?

### Expected behavior

clustering should be on the table.

### Observed behavior

no change in post deployment.

### Terraform Configuration

```hcl
took the reference from : https://cloud.google.com/bigquery/docs/reference/rest/v2/tables

TABLENAME_CLUSTERING.json
{
  "fields": [
    "FIELD_NAME"
  ]
}

when deployed, there is no change on the table.

below is the syntax for the clusting parameters in tables in main.tf file 

clustering         = try(t.clustering && t.clustering == true ? jsondecode(file("${path.module}/${upper(t.id)}_CLUSTERING.json"))[0] : null, null),
```

### Terraform Version

```sh
version = "~> 1.0.8"
```

**Comment by rmanktalacolt** [comment]
parameter for clustering is clearly listed in terraform documentation: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#clustering
